u6-spas/src/views/Converter.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+ <table>
+   <tr>
+     <td><input v-model="input" type="number"></td>
```

Comment:
คุณควรใช้ v-model.number ไม่เช่นนั้นอินพุตจะเป็น string
You should use v-model.number, otherwise the input will be a string.